container/Dockerfile

```diff
@@ -38,19 +38,19 @@ ENTRYPOINT [ "/opt/dirac/dockerEntrypoint.sh" ]
 # Create self-signed host certificate from auto-generated CA
 WORKDIR /opt/dirac/etc/grid-security
 RUN source /opt/dirac/bashrc && openssl genrsa -out hostkey.pem 2048
-RUN curl -L -o openssl_config https://raw.githubusercontent.com/fstagni/DIRAC/devDocsDocker-1/tests/openssl_config_DIRACDockerDevBox && \
+RUN curl -L -o openssl_config https://raw.githubusercontent.com/DIRACGrid/DIRAC/integration/tests/openssl_config_DIRACDockerDevBox && \
     source /opt/dirac/bashrc && \
     openssl req -new -x509 -key hostkey.pem -out hostcert.pem -days 365 -config openssl_config && \
     cp hostcert.pem certificates/ && \
     cp hostkey.pem certificates/
 
 # Now generating the user credentials, which would need to be used for connecting to DIRAC components running here
 WORKDIR /opt/dirac/user
-RUN cp /opt/dirac/etc/grid-security/openssl_config .
-RUN sed -i 's/DIRACDockerDevBox/ciuser/g' openssl_config
+RUN curl -L -o openssl_config_usr https://raw.githubusercontent.com/DIRACGrid/DIRAC/integration/tests/openssl_config_DIRACDockerDevBox_usr
+RUN sed -i 's/DIRACDockerDevBox/ciuser/g' openssl_config_usr
 RUN source /opt/dirac/bashrc && \
     openssl genrsa -out client.key 1024 && \
-    openssl req -key client.key -new -out client.req -config openssl_config
+    openssl req -key client.key -new -out client.req -config openssl_config_usr
 # This is a little hack to make OpenSSL happy...
```

Comment:
I think if you use -CAcreateserial option in the subsequent openssl x509 -req ... command, then you do not need this hack